docs/articles/blog/2019-2-7-testcafe-v1-0-0-released.md

```diff
@@ -0,0 +1,162 @@
+    "takeScreenshotsOnFails": true,
+    "videoPath": "/home/user/tests/videos/",
+    "pageLoadTimeout": 1000,
+    "hostname": "host.mycorp.com"
+    // and more
+}
+```
+
+Keep the configuration file in the project's root directory from which you run TestCafe.
+
+Settings you specify when you launch tests from the command line and programming interfaces override settings from `.testcaferc.json`.
+
+See [Configuration File](https://devexpress.github.io/testcafe/documentation/using-testcafe/configuration-file.html) for more information.
+
+### ⚙ Live Mode ([#3215](https://github.com/DevExpress/testcafe/issues/3215))
+
+We have integrated the [testcafe-live](https://github.com/DevExpress/testcafe-live) module into our main code so you can now use the new [live mode](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/live-mode.html).
+
+Live mode keeps the TestCafe process and browsers opened the whole time you are working on tests. Changes you make in code immediately restart the tests. That is, live mode allows you to see test results instantly. See [How Live Mode Works](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/live-mode.html#how-live-mode-works).
+
+Use the [-L (--live)](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-l---live) flag to enable live mode from the command line interface.
+
+```sh
+testcafe chrome tests/test.js -L
+```
+
+In the API, create a [live mode runner](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/livemoderunner.html) with the [testcafe.createLiveModeRunner](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/testcafe.html#createlivemoderunner) function and use it instead of a [regular test runner](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html).
+
+```js
+const createTestCafe = require('testcafe');
+let testcafe         = null;
+
+createTestCafe('localhost', 1337, 1338)
+    .then(tc => {
+        testcafe         = tc;
+        const liveRunner = testcafe.createLiveModeRunner();
+        return liveRunner
+            .src('tests/test.js')
+            .browsers('chrome')
+            .run();
+    })
+    .then(() => {
+        testcafe.close();
+    });
+```
+
+### ⚙ Custom Reporter API Enhancements (Part of [#2753](https://github.com/DevExpress/testcafe/issues/2753); [Pull Request](https://github.com/DevExpress/testcafe/pull/3177))
+
+* You can now access warnings that appeared during the test run from the [reportTestDone](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/reporter-methods.html#reporttestdone) method. Use the `warnings` property of the [testRunInfo](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/reporter-methods.html#testruninfo-object) object.
```

Comment:
Use the [testRunInfo](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/reporter-methods.html#testruninfo-object) object's `warnings` property

Reply:
But, use it to do what?

Reply:
To do what is written in the first sentence - 'access warnings'. Will adding 'to do this' help?

*You can now access warnings that appeared during the test run from the [reportTestDone](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/reporter-methods.html#reporttestdone) method. Use the `warnings` property of the [testRunInfo](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/reporter-methods.html#testruninfo-object) object to do this.*